znnl/models/jax_model.py

```diff
@@ -58,27 +60,39 @@ def __init__(
         optimizer : Callable
                 optimizer to use in the training. OpTax is used by default and
                 cross-compatibility is not assured.
-        input_shape : tuple
-                Shape of the NN input.
+        input_shape : Optional[tuple]
+                Shape of the NN input. Required if no pre-built model is passed.
         seed : int, default None
                 Random seed for the RNG. Uses a random int if not specified.
-        ntk_batch_size : int, default 10
+        ntk_batch_size : Optional[int], default 10
```

Comment:
Thanks for these updates. They are very helpful but also tedious to actually do, so I appreciate it.